### PR TITLE
refactor: refactor container update diskquota type

### PR DESCRIPTION
**Pull request by rudyfly**
### Ⅰ. Describe what this PR did
refactor container update diskquota type, change it from `string` to
`map[string]string`, make it consistent with container create api.

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅴ. Special notes for reviews

Signed-off-by: Rudy Zhang <rudyflyzhang@gmail.com>

### CODE REVIEW

File: apis/plugins/ContainerPlugin.go [hunk 1]
```diff
@@ -16,4 +16,8 @@ type ContainerPlugin interface {
 	// and if this endpoint should enable resolver and a map which will be used as generic params to create endpoints of
 	// this container
 	PreCreateEndpoint(string, []string) (priority int, disableResolver bool, genericParam map[string]interface{})
+
+	// PreUpdate defines plugin point where receives an container update request, in this plugin point user
```
Comment (HusterWan) [inline apis/plugins/ContainerPlugin.go]: %s/an/a 😆

File: cli/update.go [hunk 2]
```diff
@@ -52,7 +51,7 @@ func (uc *UpdateCommand) addFlags() {
 	flagSet.StringSliceVarP(&uc.env, "env", "e", nil, "Set environment variables for container")
 	flagSet.StringSliceVarP(&uc.labels, "label", "l", nil, "Set label for container")
 	flagSet.StringVar(&uc.restartPolicy, "restart", "", "Restart policy to apply when container exits")
-	flagSet.StringVar(&uc.quota, "quota", "", "Update disk quota for container")
+	flagSet.StringSliceVar(&uc.diskQuota, "disk-quota", nil, "Update disk quota for container")
```
Comment (allencloud) [inline cli/update.go]: I think we should add more details here in the command line usage. For exapmle, how to unset the original diskquota @rudyfly

Comment (HusterWan) [inline cli/update.go]: Actually, in cli/common_flag we have already defined the `diskQuota` , so if we change the type to slice, no need define here.

**Comment by HusterWan** [comment]
LGTM

**Pull request merged**